lib/scrape_rubinius.bash

```diff
@@ -0,0 +1,24 @@
+RBX_BASE_URL=https://rubinius-releases-rubinius-com.s3.amazonaws.com
+
+scrape_rubinius() {
+  local def filename num_components pkg rbx sha
+
+  for pkg in $(curl "$RBX_BASE_URL/index.txt" 2>/dev/null | grep -e '.tar.bz2$'); do
+    rbx="${pkg%.tar.bz2}"
+    sha=$(curl "$RBX_BASE_URL/$pkg.sha512" 2>/dev/null)
+    def=$(cat <<DEF
```

Comment:
Saving `def` to a local variable is clumsy and unnecessary. You should pipe it directly to a file instead.